docs/architecture/types/README.md

```diff
@@ -200,6 +257,41 @@ Output<T>>`). In this manner, input properties may accept either plain values
 defined in the program outright, or values which arise from the outputs of other
 resources.
 
+(property-paths)=
```

Comment:
There's also a `property-path.md`.  We should combine that with this probably.

Reply:
Yes, I've taken this from there. I've left the file there for now as it has a few implementation details (e.g. Go functions for working with property paths) that I want to flesh out/find a home for.

Reply:
Sure :+1: